linalg/sparsemat.cpp

```diff
@@ -687,7 +687,12 @@ void SparseMatrix::AddMult(const Vector &x, Vector &y, const double a) const
       // Allocate kernel space. Buffer is shared between different sparsemats
       size_t newBufferSize = 0;
 
-#if CUDA_VERSION > 10010 || CUDA_VERSION == 10010
+#if CUDA_VERSION > 11020 || CUDA_VERSION == 11020
```

Comment:
Is there a reason to not use `CUDA_VERSION >= 11020` here and in other places?

Reply:
@adam-sim-dev  we will replace all these instances with `>=` as @v-dobrev suggested. Please speak up if you have any concerns.

Reply:
☝️  implemented in  https://github.com/mfem/mfem/pull/2200/commits/914c334a8b6342c7b3abf298a91552198bd3ce48